base/linalg/bidiag.jl

```diff
@@ -154,17 +172,38 @@ for func in (:A_ldiv_B!, :Ac_ldiv_B!, :At_ldiv_B!)
         B
     end
 end
-for func in (:A_ldiv_Bt!, :Ac_ldiv_Bt!, :At_ldiv_Bt!)
+
+function A_ldiv_Bt!(A::Union(Bidiagonal, AbstractTriangular), B::AbstractMatrix)
+    mA,nA = size(A)
+    tmp = similar(B[1,:])
+    tmp = tmp.'
+    m, n = size(B)
+    if mA != n || nA != m
+        throw(DimensionMismatch())
+    end
+    for i = 1:size(B, 1)
+        copy!(tmp, B[i,:])
+        A_ldiv_B!(A, tmp)
+        B[i,:] = tmp.'
+    end
+    B.'
```

Comment:
I think that the necessity of this line makes it clear that the `..._Bt/c!` methods should simply do the explicit transposition of `B`, e.g. `A_ldiv_Bt(A,B)=A_ldiv_B!(A,transpose(B))`. Note that there is no exclamation mark for the first method. This must be the case for all solves: it doesn't make sense to pretend a in-place operations when the result has to be copied anyway. Hence, you should update https://github.com/JuliaLang/julia/blob/b8ecc0e191350a79b9dca3e0ff5f48c75707623d/base/operators.jl#L161 and https://github.com/JuliaLang/julia/blob/b8ecc0e191350a79b9dca3e0ff5f48c75707623d/base/operators.jl#L164 with version that only transposes `B` and remove any of these definitions from this file. Does this make sense?